src/chain.js

```diff
@@ -11,19 +11,25 @@ var map = require('./map');
  *
  * Dispatches to the `chain` method of the second argument, if present,
  * according to the [FantasyLand Chain spec](https://github.com/fantasyland/fantasy-land#chain).
+ * Treats the second argument as a FantasyLand Chain, when the the second argument is a function
+ * of the form `a -> x -> b`.
  *
  * @func
  * @memberOf R
  * @since v0.3.0
  * @category List
  * @sig Chain m => (a -> m b) -> m a -> m b
+ * @sig (x -> a) -> (a -> x -> b) -> (x -> b)
```

Comment:
> It didn't help that the obviously bright @bergus seemed to pull this from nowhere and just expect it to work.

I imagine that @bergus is familiar with Haskell. It's difficult to guess the argument order of `R.chain` based on `(>>=)`, since the latter is usually used infix.